docs/forms-apps.md

```diff
@@ -27,10 +28,54 @@ const OneBlink = require('@oneblink/sdk')
 const options = {
   accessKey: '123455678901ABCDEFGHIJKL',
   secretKey: '123455678901ABCDEFGHIJKL123455678901ABCDEFGHIJKL',
+  tenant: 'ONEBLINK',
 }
 const formsAppsSDK = new OneBlink.FormsApps(options)
 ```
 
+## `verifyJWT()`
+
+Verify a JWT and return its result
+
+### Example
+
+```javascript
+formsAppsSDK
```

Comment:
Can we also support passing in the `Authorization` header value which will be `Authorization: 'Bearer {token}'`

Would be good to have that or here in the example aswell.

```js
const token = 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c'
// or
const token = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c'
formsAppsSDK
  .verifyJWT(token)
```